drizzlepac/devutils/disp_sources_by_flag.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+"""
```

Comment:
comparison_tools/compare_sourcelist_flagging.py is just a version of drizzlepac/devutils/disp_sources_by_flag.py modified to with two user-specified sourcelists, rather than just one. The two scripts do share some common code, but there enough modifications to the code in comparison_tools/compare_sourcelist_flagging.py to keep it as a separate script